src/Microsoft.Health.Fhir.Core/Configs/VersioningConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Fhir.ValueSets;
+
+namespace Microsoft.Health.Fhir.Core.Configs
+{
+    public class VersioningConfiguration
+    {
+        public string Default { get; set; } = ResourceVersionPolicy.Versioned;
```

Comment:
Do we want to verify that this string is a valid value from the enumeration?  I am not sure if we check values in the appsettings.json in general, or if we want to do so in this case.  Perhaps if it is an invalid value, we default to "versioned"?

Reply:
Good point, I was thinking about this as I started working on some tests for this... It doesn't look like we validate the values of any other app settings, but I am happy to put in validation if this scenario calls for it. Thoughts @brendankowitz?

Reply:
As discussed, I created a user story for validating all app settings (please see [AB#85362](https://microsofthealth.visualstudio.com/Health/_workitems/edit/85362)).